### PR TITLE
Cookies disclaimer

### PR DESCRIPTION
**Pull request by vbulant**
https://github.com/actum/gulp-dev-stack/issues/82
@zoltan-kovac works on *accordion* module so let’s keep the issue open

### CODE REVIEW

**Comment by vbulant** [comment]
CookieLaw fn is definitely useful, but it’s not the best example of factory functions (in fact, it’s not a factory function :).

Hope accordion will be better example.

File: src/styles/critical.scss [hunk 1]
```diff
@@ -9,3 +9,7 @@
 h1 {
     color: purple;
 }
+
+.hidden {
```
Comment (janpanschab) [inline src/styles/critical.scss]: Use `.u-hidden` as a utility class or create cookie-law component with BEM.

File: src/tpl/layouts/default.nunj [hunk 2]
```diff
@@ -41,6 +41,8 @@
     {% endif %}
 </head>
 <body>
+    {% include "../partials/cookie-law.nunj" %}
```
Comment (janpanschab) [inline src/tpl/layouts/default.nunj]: I don’t like partials here. It's time to remove it. It tends to use structure folder as container of all partials. I would suggest `tpl/component(s)/cookie-law.nunj`.

File: src/tpl/partials/cookie-law.nunj [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<p class="alert alert-info pos-f-t hidden" id="cookie-law" role="alert">
```
Comment (janpanschab) [inline src/tpl/partials/cookie-law.nunj]: Indentation 4 instead 2.

**Review by janpanschab — CHANGES_REQUESTED** [review]

**Review by janpanschab — APPROVED** [review]

**Pull request merged**